--- libdsm-0.4.0/bin/meson.build.orig	2022-03-03 11:05:14.574187500 +0100
+++ libdsm-0.4.0/bin/meson.build	2022-04-01 06:26:12.527690929 +0200
@@ -1,19 +1,23 @@
 executable('dsm', 'dsm.c', 
   include_directories: [includes],
+  install: true,
   link_with: libdsm
 )
 
-executable('discover', 'discover.c',
+executable('dsm_discover', 'discover.c',
   include_directories: [includes],
+  install: true,
   link_with: libdsm
 )
 
-executable('inverse', 'inverse.c',
+executable('dsm_inverse', 'inverse.c',
   include_directories: [includes],
+  install: true,
   link_with: libdsm
 )
 
-executable('lookup', 'lookup.c',
+executable('dsm_lookup', 'lookup.c',
   include_directories: [includes],
+  install: true,
   link_with: libdsm
 )
