--- libdsm-0.4.0/doc/Doxyfile.in.orig	2022-03-03 11:05:14.574187500 +0100
+++ libdsm-0.4.0/doc/Doxyfile.in	2022-04-01 06:18:53.121670869 +0200
@@ -1009,7 +1009,7 @@ GENERATE_HTML          = YES
 # The default directory is: html.
 # This tag requires that the tag GENERATE_HTML is set to YES.
 
-HTML_OUTPUT            = html
+HTML_OUTPUT            = ../html
 
 # The HTML_FILE_EXTENSION tag can be used to specify the file extension for each
 # generated HTML page (for example: .htm, .php, .asp).
--- libdsm-0.4.0/meson.build.orig	2022-04-01 06:23:05.352256769 +0200
+++ libdsm-0.4.0/meson.build	2022-04-01 06:23:13.973876755 +0200
@@ -271,7 +271,7 @@ if doxygen.found()
     configuration: doxygen_data,
     install: false)
   doc_dir = join_paths(get_option('datadir'), 'doc', 'libdsm')
-  run_target('doc',
+  custom_target('doc',
              command: [doxygen, doxyfile],
              input: doxyfile,
              output: 'html',
--- libdsm-0.4.0/bin/meson.build.orig	2022-03-03 11:05:14.574187500 +0100
+++ libdsm-0.4.0/bin/meson.build	2022-04-01 06:26:12.527690929 +0200
@@ -1,19 +1,23 @@
 executable('dsm', 'dsm.c', 
   include_directories: [includes],
+  install: true,
   link_with: libdsm
 )
 
-executable('discover', 'discover.c',
+executable('dsm_discover', 'discover.c',
   include_directories: [includes],
+  install: true,
   link_with: libdsm
 )
 
-executable('inverse', 'inverse.c',
+executable('dsm_inverse', 'inverse.c',
   include_directories: [includes],
+  install: true,
   link_with: libdsm
 )
 
-executable('lookup', 'lookup.c',
+executable('dsm_lookup', 'lookup.c',
   include_directories: [includes],
+  install: true,
   link_with: libdsm
 )
